documentation/production-checklist.md

```diff
@@ -0,0 +1,62 @@
+
+## Multiple replicas
+By default the template deploys only 1 replica for each [kubernetes deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/). This is not sufficient for production as if the container is unavailable, there is no other replica to serve the requests. It may be unavailable because of high usage or simply because the cluster is moving the container to another node. This will happen when the cluster version is updated.
+
+Use at least 2 [replicas](https://github.com/DFE-Digital/terraform-modules/blob/04895b849cd5124e615b4e6b1850c0d918d4d081/aks/application/variables.tf#L32) or as many as required by [performance testing](#performance-testing).
+
+## Database plan
+The template deploys a default plan for [postgres](https://github.com/DFE-Digital/terraform-modules/blob/83801213853ed1e4b4bdcb8d36773c8683ff010f/aks/postgres/variables.tf#L82) and [redis](https://github.com/DFE-Digital/terraform-modules/blob/83801213853ed1e4b4bdcb8d36773c8683ff010f/aks/redis/variables.tf#L63-L71).
+
+It may be sufficient for the test environments, but it may not offer enough CPU, memory or network bandwidth for production. [Performance testing](#performance-testing) will help determine the right plans.
+
+Note that for redis, all `azure_family`, `azure_sku_name` and `azure_capacity` must be changed jointly. Check [terraform postgres documentation](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/postgresql_flexible_server) for the allowed values.
+
+## High availability
+Each Azure region provides multiple [availability zones](https://learn.microsoft.com/en-us/azure/reliability/availability-zones-overview). The kubernetes cluster is deployed across 3 zones so in case one is failing, the workload continues on the 2 others.
+
+The same should be applied to database clusters. For postgres, set `azure_enable_high_availability` to true. For redis, use a Premium plan.
+
+Note the cost is doubled for postgres, and [much higher](https://azure.microsoft.com/en-gb/pricing/details/cache/) for redis, so this should be used carefully.
+
+## Performance testing
+Simulate load from user traffic to determine the right number of instances and the database plan. This should cover the most typical user journeys. We recommend [K6](https://k6.io/) as it can be deployed to the cluster to minimise latency. Check the example in [teacher pay calculator](https://github.com/DFE-Digital/teacher-pay-calculator/tree/main/load_testing).
+
+If time is short or user traffic is expected to be low, make sure to monitor the application and database usage after launch, and everytime there is a new significant feature. And be ready to scale up.
+
+## Postgres and redis monitoring
```

Comment:
Add postgres backup should be in place for production before launch?

Reply:
Added